src/api/goods.ts

```diff
@@ -5,15 +5,24 @@ const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/good
 
 export function getAll(): Promise<Good[]> {
   return fetch(API_URL)
-    .then(response => response.json());
+    .then(response => {
+      if (!response.ok) {
+        throw new Error `${response.status} - ${response.statusText}`;
+      }
```

Comment:
There is some type error after adding 'new Error', and I don't know how to fix it. https://prnt.sc/9Up3j2wdJUTP